docs/manuals/cli/run.md

```diff
@@ -0,0 +1,118 @@
+# Run application in Taipy CLI
+
+Assume that you have a "main.py" file containing your Taipy application.
+To start the Taipy application, you can run:
+
+```console
+$ taipy run main.py
+```
+
+!!! note
```

Comment:
=>

"!!! note "Using the 'python' command"

Reply:
```suggestion
!!! note "Using the 'python' command"
```